### ISSUE TITLE
Add link to polymorphic guide from query guide

### ISSUE THREAD

**Issue by paulcsmith** [luckyframework/website#112]
* Cover validations, using with save operations, preloading  and querying.

See docs https://github.com/luckyframework/avram/blob/master/src/avram/polymorphic.cr#L3

**Comment by jwoertink** [comment]
This is in addition to these guides? https://www.luckyframework.org/guides/database/models#polymorphic-associations

**Comment by paulcsmith** [comment]
Ohhh. For some reason I was looking in the querying guide! I think part of the problem is the search index didn't update yet so I didn't see it in search. This can be closed :) My bad

**Comment by jwoertink** [comment]
We could always add a link in the query guide that points over to this. Like maybe where associations are...

**Comment by paulcsmith** [comment]
Yeah I think that would be a good idea. We do that in a few other places too and I think it works well
